### PR TITLE
switch to npm

### PR DESCRIPTION
**Pull request by TimaJuma**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,9 @@
     "type": "git",
     "url": "https://github.com/neofinancial/eslint-config-neo.git"
   },
+  "engines": {
+    "npm": ">=8.0.0"
```
Comment (TimaJuma) [inline package.json:14]: or `"npm": ">=8.3.0"` ?

Comment (rogersnick) [reply package.json:14]: Either is probably fine

**Review by rogersnick — APPROVED** [review]

**Comment by TimaJuma** [comment]
Did not touch `README` as it refers to the installation of this library, or we can add additional commands for `npm` as well in the description

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.7.0 (December 22, 2021)
```
Comment (iansu) [inline CHANGELOG.md:3]: This doesn't need a version bump because there's nothing new to publish. It's just an internal change.

**Review by iansu — CHANGES_REQUESTED** [review]

**Review by rogersnick — APPROVED** [review]

**Review by iansu — APPROVED** [review]

**Pull request merged**